Tests/js/views/fields/ez-richtext-editview.html

```diff
@@ -14,9 +14,9 @@
     <div class="ez-richtext-editor {{editableClass}}" contenteditable="false">
         {{{ xhtml }}}
     </div>
+    <button class="ez-richtext-save-and-return">Save and return</button>
     <p class="ez-editfield-error-message"></p>
     <button class="ez-richtext-switch-focus">Focus</button>
-    <div class="ez-focusmodeactionbar-container"></div>
```

Comment:
more cleanup needed here as, the model "ez-richtextfocusmodebarview" does not exist anymore, "ez-barview" and "ez-buttonactionview" are not needed.